### ISSUE TITLE
Sev1 Accessibilty bugs, Please looking for solution on these reporetd accessibility bugs.

### ISSUE THREAD

**Issue by kotapavan07** [xdan/datetimepicker#813]
I have been using your Datetime picker in my webportal, It works great for us thank you,
But Last month my webportal went through the Accessibility checks and they reported bugs and requested us fix following scenarios. Could you please help us to fix these accessibility bugs.

**1. On invoking "issue start date and time" edit box we can be able to navigate on the calendar and we can be able to select the elements in the calendar pane

2. Upon Invoking the “issue start date & time” in calendar “previous, next, Home, Downward and Upward buttons Should be clearly visible in High Contrast Aquatic theme.

3. The minimum color contrast ratio for the non-selected elements should be more than 4.5:1, the minimum color contrast ratio for the text is 4.5:1
 
4. On giving show numbers command, numbers should be shown for all the elements in calendar.**

```
<div class="col-md-6">
      <label class="font-weight-bold" for="IssueDate">Issue Start Date & Time</label>
       @Html.TextBoxFor(m => m.IssueDate, new { @class = "form-control", type = "text", value = "" })
</div>

//Javascript
        $('#IssueDate').datetimepicker({
            format: 'm/d/Y H:i:s',
            // row class was added to scale it properly at 400% zoom level to a single row
			className: 'row'
        });
```

Thank you
